unix.go

```diff
@@ -0,0 +1,11 @@
+package adb
+
+/*
+Exported Access function calls private access function.
+Implementation for private access function is provided in
+access_win.go for windows and
+access_unix.go for unix
+*/
+func Access(path string) error {
```

Comment:
`access` is a unix-specific term. Since this is intended to be cross-platform, how about calling it something like `IsExecutable`?